src/components/main-layout.module.scss

```diff
@@ -5,15 +5,28 @@
 
   .header {
     height: 50px;
-    background-color: orange;
+    background-color: #6e819e;
```

Comment:
don't use constants for color

the better way extract all them to variables to have control on project pallete